runtime/core/decoder/ctc_prefix_beam_search.cc

```diff
@@ -186,4 +186,42 @@ void CtcPrefixBeamSearch::Search(const torch::Tensor& logp) {
   }
 }
 
+void CtcPrefixBeamSearch::FinalizeSearch() {
+  UpdateFinalContext();
+}
+
+void CtcPrefixBeamSearch::UpdateFinalContext() {
+  if (context_graph_ == nullptr) return;
+  CHECK_EQ(hypotheses_.size(), cur_hyps_.size());
+  CHECK_EQ(hypotheses_.size(), likelihood_.size());
+  // The context that has not yet the end boudary does not set to start state
```

Comment:
Please refinte the comment.
We should backoff the context score/state when the context is not fully matched at the last frame.